exercises/practice/pangram/.docs/instructions.md

```diff
@@ -1,9 +1,8 @@
 # Instructions
 
-Determine if a sentence is a pangram. A pangram (Greek: παν γράμμα, pan gramma,
-"every letter") is a sentence using every letter of the alphabet at least once.
-The best known English pangram is:
-> The quick brown fox jumps over the lazy dog.
+Your task is to figure out if a sentence is a pangram.
 
-The alphabet used consists of ASCII letters `a` to `z`, inclusive, and is case
-insensitive.
+A pangram is a sentence using every letter of the alphabet at least once.
```

Comment:
```suggestion
A pangram is a sentence using every letter of an alphabet at least once.
```

This should be "an" otherwise being specific about it using the English alphabet doesn't match this sentence well.